src/xclim/indices/run_length.py

```diff
@@ -465,11 +507,13 @@ def windowed_max_run_sum(
         Total number of `True` values part of a consecutive runs of at least `window` long.
     """
     if window == 1 and freq is None:
-        out = rle(da, dim=dim, index=index).max(dim=dim)
+        # using _cumsum_reset_xr instead of rle to be able to handle a float
+        out = _cumsum_reset_xr(da, dim=dim, index=index, reset_on_zero=True).max(dim=dim)
```

Comment:
Why not use the generic `_cumsum_reset` ?

Reply:
The generic `_cumsum_reset` may use the fast track (`_cumsum_reset_np`) depending on the chunking. The fast track explicitly assumes that we have 1's and 0', and we don't want tha. Here, we use `_cumsum_reset_xr`, which allows this kind of behaviour:
2 2 0 0 2 2 2 0 0  -> 2 4 0 0 2 4 6 0 0 

Perhaps having the _cumsum_reset function that should be used only with 1's and 0's is not so intuitive. Again, this is because the fast track exploits the idea of having only 1's and 0's.

Reply:
You could call the function `binary_cumsum_with_reset`.